src/app/app.tsx

```diff
@@ -16,8 +17,15 @@ m.route(document.body, '/logs', {
             </Layout>
         ),
     },
+    '/logs/create': {
```

Comment:
There's something weird about the order of these routes. If you put /logs/create below /logs/:id, it always tries to route the 'create' part as an id, which is invalid. Might need to refactor if this becomes problematic.